Foundation/src/double-to-string.cc

```diff
@@ -56,15 +56,15 @@ bool DoubleToStringConverter::HandleSpecialValues(
     StringBuilder* result_builder) const {
```

Comment:
we do not fix embedded dependencies directly. Instead, you should report it upstream

Reply:
Removed that commit